app/templates/components/session-card.hbs

```diff
@@ -0,0 +1,43 @@
+<div class="{{if isWide 'event wide ui grid row'}}">
```

Comment:
@geekyd Why is `isWide` required, where will session cards be used in non wide mode?